integration/push_test.go

```diff
@@ -111,8 +111,14 @@ func TestPushAlwaysMostRecentlyCreatedIndex(t *testing.T) {
 				index := buildIndex(sh, regConfig.mirror(tc.image), withMinLayerSize(opt.minLayerSize), withSpanSize(opt.spanSize), withOCIArtifactRegistrySupport)
 				index = strings.Split(index, "\n")[0]
 				out := sh.O("soci", "push", "--user", regConfig.creds(), regConfig.mirror(tc.image).ref, "-q")
-				pushedIndex := strings.Trim(string(out), "\n")
-
+				output := strings.Split(string(out), "\n")
+				output = output[:len(output)-1]
+				var pushedIndex string
+				if len(output) == 1 {
+					pushedIndex = strings.TrimSpace(output[0])
+				} else {
+					pushedIndex = strings.TrimSpace(output[1])
+				}
```

Comment:
why is this?

Reply:
We push soci indices twice with the `-q` flag, meaning along with the index digest a  'WARN' will still be printed on second push. This checks if the length of the output is longer than 1 (IE when there is WARN message) and if so grabs just the index digest.

Reply:
Yup didnt even notice that haha, thanks I'll change it